### ISSUE TITLE
AttributeError: 'FreeTypeFont' object has no attribute 'getsize_multiline'

### ISSUE THREAD

**Issue by JaxsonYorke** [nobrien97/FortuneBot#2]
So I think the problem here is something with pillow or the font, when I try to install/update/upgrade pillow with python3 it gives me a build error for`/Pillow-8.3.2` and when I install with python it installs `Pillow-6.2.2` what is the proper version? Also, should I be using `python3` or `python`?
Installing pillow with python
![image](https://user-images.githubusercontent.com/38132640/131903495-dd861d90-cd66-45ee-bc0b-786f78d0d9e5.png)
Installing pillow  with python3
![image](https://user-images.githubusercontent.com/38132640/131903613-24d1fec9-6245-4dbf-a98b-097336f0a112.png)
running with python3
![image](https://user-images.githubusercontent.com/38132640/131903782-0b6b4658-e76a-4f9a-9dc8-df225f537afa.png)
also, when I run main.py with python it gives me an error about async def
![image](https://user-images.githubusercontent.com/38132640/131904396-e8f5ab19-8272-4c34-9bd3-c4065646ee8b.png)

also, when I try to use your command in the readme to install discord api it gives me an error but works when I use  `python3 -m pip install -U discord.py` it works

installing discord with python
![image](https://user-images.githubusercontent.com/38132640/131905185-b619f0b4-9206-45f0-b733-39e2ed97c2a9.png)
installing discord with python3
![image](https://user-images.githubusercontent.com/38132640/131905324-d1facbab-f483-4a8b-9742-e7cc7e24afab.png)

**Comment by JaxsonYorke** [comment]
Sorry if this is a lot, I'm happy to help wherever I can

**Comment by JaxsonYorke** [comment]
After looking more in depth (on Reddit) I think that attribute `getsize_multiline` is only in pillow for python 2.7, and I'm getting errors when trying to install pillow from `python` get that as said above
reddit thread: https://www.reddit.com/r/learnpython/comments/343afg/when_i_try_to_call_pilimagefontimagefontgetsize_i/

**Comment by nobrien97** [comment]
Yeah, so in many Linux distros `python` by default runs python 2, which is quite a different language to python 3. I forgot that I had set `python` as an alias of `python3` in my `.bashrc`, hence why it isn't working for you. I've updated the readme to be more clear. I just updated Pillow and it still works fine. You'll need to use python3. From the error, for some reason you're missing a dependency.... try installing `libjpeg` and/or `libjpeg-dev`
See here: https://github.com/python-pillow/Pillow/issues/5042
